blockchain/emulator_test.go

```diff
@@ -59,5 +63,123 @@ func Test_TranslateAddress(t *testing.T) {
 		out := translateAddresses(in[0])
 		assert.Equal(t, string(in[1]), string(out))
 	}
+}
+
+// Test_NewEmulator tests creating a large number of new accounts and validates corresponding storage addresses
+func Test_NewEmulator(t *testing.T) {
+	emu, err := newEmulator()
+	assert.NoError(t, err)
+
+	var accountList []*flowsdk.Account
+
+	const testAccounts int = 1000
+
+	fmt.Println("Creating", testAccounts, "new accounts...")
+	for i := 0; i < testAccounts; i++ {
+		account, _, _, err := emu.createAccount()
+		assert.NoError(t, err)
+		accountList = append(accountList, account)
+	}
+
+	fmt.Println("Validating account storage...")
+	for i := 0; i < testAccounts; i++ {
+		_, accountStorage, err := emu.getAccount(accountList[i].Address)
+		assert.NoError(t, err)
+		assert.Equal(t, accountStorage.Account.Address.String(), accountList[i].Address.String())
+		assert.Equal(t, accountStorage.Account.Address.Hex(), accountList[i].Address.Hex())
+		assert.Equal(t, accountStorage.Account.Address.Bytes(), accountList[i].Address.Bytes())
+	}
+}
+
+// Test_DeployEmptyContract tests deployment of an empty contract
```

Comment:
It's good practice to group common tests and then have subgroups, so in case this could be top-level Test_DeployContract test and then include subtests. Example: https://github.com/onflow/flow-playground-api/blob/improvement/emulator-refactor/blockchain/projects_test.go#L94